src/modules/admin/components/Permissions/Permissions.tsx

```diff
@@ -0,0 +1,203 @@
+import React, { useCallback, useMemo, useState } from 'react';
+import { defineMessages, FormattedMessage } from 'react-intl';
+import sortBy from 'lodash/sortBy';
+import { ColonyRole, ROOT_DOMAIN_ID } from '@colony/colony-js';
+
+import Heading from '~core/Heading';
+import { Select, Form } from '~core/Fields';
+import { Table, TableBody, TableCell } from '~core/Table';
+import Button from '~core/Button';
+import { useDialog } from '~core/Dialog';
+import ExternalLink from '~core/ExternalLink';
+import { useTransformer } from '~utils/hooks';
+import { Colony } from '~data/index';
+
+import { getAllUserRolesForDomain } from '../../../transformers';
+import UserListItem from '../UserListItem';
+import UserPermissions from './UserPermissions';
+import ColonyPermissionsAddDialog from './ColonyPermissionsAddDialog';
+
+import styles from './Permissions.css';
+
+const DOMAINS_HELP_URL = 'https://help.colony.io/';
+
+const MSG = defineMessages({
+  title: {
+    id: 'admin.Permissions.title',
+    defaultMessage: `Permissions{domainLabel, select,
+      root {}
+      other {: {domainLabel}}
+    }`,
+  },
+  labelFilter: {
+    id: 'admin.Permissions.labelFilter',
+    defaultMessage: 'Filter',
+  },
+  addRole: {
+    id: 'admin.Permissions.addRole',
+    defaultMessage: 'Add Role',
+  },
+  permissionInParent: {
+    id: 'admin.Permissions.permissionInParent',
+    defaultMessage: '*Permission granted via parent domain. {learnMore}',
+  },
+  learnMore: {
+    id: 'admin.Permissions.learnMore',
+    defaultMessage: 'Learn more',
+  },
+});
+
+interface Props {
+  colony: Colony;
+}
+
+const displayName = 'admin.Permissions';
+
+const Permissions = ({ colony: { colonyAddress, domains }, colony }: Props) => {
```

Comment:
This component is not needed and was already removed in #2295

Don't add it back, please remove it from your branch as well